src/experimental/tools/VideoThumbnailLoader/video_thumbnail_loader.ts

```diff
@@ -184,6 +184,7 @@ export default class VideoThumbnailLoader {
         "video",
         loader.video,
         null,
+        null,
```

Comment:
This function has 6 arguments, it's a bit difficult to follow,
we might consider grouping some arguments into an object.
Other solution could be adding the argument name to understand the intention of thoses null
```js
createSegmentFetcher(
        ...
        /* cdnPrioritizer*/ null,
        /* cmcdDataBuild*/ null,

```

Reply:
You're right, I guess the object could be good.

`cdnPrioritizer` and `cmcdDataBuilder` are hard to justify having an order between one another when it is multiple ordered arguments.

Reply:
Done